pkg/proxy/winkernel/OWNERS

```diff
@@ -1,3 +1,6 @@
+approvers:
+- dineshgovindasamy
+- madhanrm
```

Comment:
@madhanrm Also add me to the list? I'd like to improve this part in next release.

Reply:
done